operator/controllers/btpoperator_controller.go

```diff
@@ -81,6 +81,15 @@ const (
 	btpOperatorServiceBinding  = "ServiceBinding"
 )
 
+const (
+	chartVersionKey       = "chart-version"
+	btpManagerConfigMap   = "btp-manager-versioning-config-map"
```

Comment:
think about changing the ConfigMap name to something like `btp-manager-versions`